docs/guides/webview-versions.md

```diff
@@ -0,0 +1,190 @@
+
+WebView2 is supported on Windows 7 and newer and comes preinstalled on Windows 11. On versions older than Windows 11 the installer generated by Tauri takes care of ensuring WebView2 is installed on the system.
+
+## WebKit
+
+Tauri uses WebKit on macOS (through [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc)) and on Linux (through webkit2gtk).
+
+Webkit version numbers are quite complicated, so below is some helpful information to understand them.
+
+<details>
+<summary>How to read WebKit version numbers</summary>
+
+WebKit version numbers are made up of 5 segments and a numeric prefix indicating the OS WebKit is built for. The prefix is called the `SYSTEM_VERSION_PREFIX` and seems to be only present for macOS and iOS builds, not for Linux. Furthermore, if the last two segments are both `0` they can be omitted; So a version like `613.2.7.0.0` would be referred to as `613.2.7`.
+
+`$(SYSTEM_VERSION_PREFIX)$(MAJOR_VERSION).$(MINOR_VERSION).$(TINY_VERSION).$(MICRO_VERSION).$(NANO_VERSION)`
+
+### `SYSTEM_VERSION_PREFIX`
+
+Here is what the `SYSTEM_VERSION_PREFIX` values mean:
+
+| macOS version | SYSTEM_VERSION_PREFIX |
+| ------------- | --------------------- |
+| 10.11         | 11                    |
+| 10.12         | 12                    |
+| 10.13         | 13                    |
+| 10.14         | 14                    |
+| 10.15         | 15                    |
+| 11.0          | 16                    |
+| 12.0          | 17                    |
+| 13.0          | 18                    |
+| sdk=iphone\*  | 8                     |
+
+### Example
+
+The webkit version shipped with Safari 15.5 on macOS Monterey (12.x) has the following version number: `17613.2.7.1.8`.
+
+| part                    | version |
+| ----------------------- | ------- |
+| `SYSTEM_VERSION_PREFIX` | 17      |
+| `MAJOR_VERSION`         | 613     |
+| `MINOR_VERSION`         | 2       |
+| `TINY_VERSION`          | 7       |
+| `MICRO_VERSION`         | 1       |
+| `NANO_VERSION`          | 8       |
+
+</details>
+
+The tables below map OS versions to WebKit versions and the corresponding Safari equivalent, so you can use sites like [caniuse](https://caniuse.com) to figure out if a specific web platform feature is supported.
+
+### macOS
+
+On macOS Tauri uses the component that comes preinstalled with macOS since version 10.10 (Yosemite). It is considered a core component and is therefore updated with the regular OS updates. This means _unsuspported_ macOS versions also **do not** receive WebKit updates.
+
+To find the Webkit version used by WKWebView you can use this command:
+
+```shell
+awk '/CFBundleVersion/{getline;gsub(/<[^>]*>/,"");print}' /System/Library/Frameworks/WebKit.framework/Resources/Info.plist
+```
+
+|             | macOS                            | WebKit           | Safari Equivalent | Notes                                                            |
```

Comment:
Reverse the sort order of the table so that most recent is at the top. Can you "merge" cells vertically in a table so that all the rows in the first column are merged?

Reply:
Maybe also do something similar for "Safari Equivalent" where they would span multiple rows?

Reply:
I'm confused, what exactly do you mean?